src/app/app.module.ts

```diff
@@ -168,7 +169,8 @@ export const myCustomSnackbarDefaults: MatSnackBarConfig = {
     ApiModule.forRoot(getApiConfig),
     CustomMaterialModule,
     RefreshAlertModule,
-    RequestsModule
+    RequestsModule,
+    ServiceWorkerModule.register('ngsw-worker.js', { enabled: environment.production })
```

Comment:
How will we test this locally?

Reply:
Also, where is this file? Is this a generic file that is installed via a dependency that goes off the manifest, or does it need to be committed?

Reply:
`ngsw-worker.js` is generated when the production build is created with `npm build.prod`; I'm not sure how we can test this locally but I can look into it. 

The committed file `ngsw-config.json` is a service worker config file that specifies the caching behaviors for `ngsw-worker.js` to include.

Reply:
From what I've read, the service worker only works in production apps